src/Illuminate/Http/Middleware/TrustProxies.php

```diff
@@ -117,8 +117,12 @@ protected function getTrustedHeaderNames()
             case Request::HEADER_X_FORWARDED_PROTO:
                 return Request::HEADER_X_FORWARDED_PROTO;
 
+            case 'HEADER_X_FORWARDED_PREFIX':
+            case Request::HEADER_X_FORWARDED_PREFIX:
+                return Request::HEADER_X_FORWARDED_PREFIX;
+
```

Comment:
Just a suggestion, but since you are targetting 9.x, this whole switch statement can be replaced with php's `match` function. What do you think?

Reply:
I like this suggestion. Should I close this PR and replace it with php's `match` function and open another PR for this?

Reply:
i can't answer that. @driesvints can help us here

Reply:
Let's focus on the change itself and we can do style changes in a different PR.